### PR TITLE
Feature/show recommended books

### PR DESCRIPTION
**Pull request by Lucassky10**

### CODE REVIEW

File: src/app/auth/interceptor.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+        const authReq = req.clone({
+            setHeaders: {
+                Authorization: 'Bearer ' + this.cookie.get('access_token')
+            }
+        });
+
+        return next.handle(authReq).pipe(tap((event: HttpEvent<any>) => {
+            if (event instanceof HttpResponse) {
+                // do something on response
```
Comment (RomainCtl) [inline src/app/auth/interceptor.ts:25]: Something ? mm.. nothing is done here

File: src/app/shared/modals/redirect-confirmation/redirect-confirmation.component.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-redirect-confirmation',
+  templateUrl: './redirect-confirmation.component.html',
+  styleUrls: ['./redirect-confirmation.component.scss']
```
Comment (RomainCtl) [inline src/app/shared/modals/redirect-confirmation/redirect-confirmation.component.ts:6]: empty file ..?

File: src/app/shared/modals/redirect-confirmation/redirect-confirmation.component.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+})
+export class RedirectConfirmationComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+
+  }
```
Comment (RomainCtl) [inline src/app/shared/modals/redirect-confirmation/redirect-confirmation.component.ts:14]: Useless function here (if empty)

**Review by RomainCtl — APPROVED** [review]

File: src/app/auth/interceptor.ts (hunk 1 above)
Comment (Lucassky10) [reply src/app/auth/interceptor.ts:25]: Haha, yes we don't need to do something yet but maybe later on

File: src/app/shared/modals/redirect-confirmation/redirect-confirmation.component.ts (hunk 2 above)
Comment (Lucassky10) [reply src/app/shared/modals/redirect-confirmation/redirect-confirmation.component.ts:6]: Yes this is a simple pop-up and these files are automatically generated by the CLI

File: src/app/shared/modals/redirect-confirmation/redirect-confirmation.component.ts (hunk 3 above)
Comment (Lucassky10) [reply src/app/shared/modals/redirect-confirmation/redirect-confirmation.component.ts:14]: Automatically generated as well

**Comment by Lucassky10** [comment]
I will merge using command line as I refactored some folders. DO NOT MERGE!

**Pull request merged**